.travis.yml

```diff
@@ -15,4 +15,4 @@ install:
 script:
   - vendor/bin/phpunit
   - bin/psalm
-  - vendor/bin/phpcs
+  - vendor/bin/php-cs-fixer fix --verbose --allow-risky=yes --dry-run
```

Comment:
I would advise to validate only files modified in given PR instead of "just everything"
See the advices on PHP CS Fixer repo/readme/website